lib/ui/recommendation.js

```diff
@@ -123,21 +123,25 @@ Recommendation.prototype = {
     this.events.publish('recommendation-mousemove', evt);
   },
   show: function(data) {
+    let payload = data.payload;
+    let ResultClass = data.resultClass;
```

Comment:
let's avoid initial uppercase letters unless it's a constructor function or an all-caps constant